tests/tensor/test_math.py

```diff
@@ -806,11 +808,7 @@ def test_basic_2(self, axis, np_axis):
         v_shape, i_shape = eval_outputs([vt.shape, it.shape])
         assert tuple(v_shape) == vt.type.shape
         assert tuple(i_shape) == it.type.shape
-        # Test values
-        v, i = eval_outputs([vt, it])
-        assert i.dtype == "int64"
-        assert np.all(v == np_max)
-        assert np.all(i == np_argm)
```

Comment:
We should still test the values are correct?

Reply:
Should we not? I felt that the tests should atleast account for the final answer in case if some easy variables